en/app.ftl

```diff
@@ -214,7 +214,7 @@ faq-question-missing-emails-answer-reason-turned-off = The alias might have forw
 faq-question-missing-emails-answer-reason-delay = { -brand-name-relay } might be taking longer than usual to forward your messages
 #   $url (url) - link to the support site
 #   $attrs (string) - specific attributes added to external links
-faq-question-missing-emails-answer-b-html = If you’re a { -brand-name-relay-premium } user struggling with any of these issues, please <a href="{ $url }" { $attrs }>contact our support team</a>.
+faq-question-missing-emails-answer-support-site-html = If you’re struggling with any of these issues, please <a href="{ $url }" { $attrs }>review our support site</a>.
```

Comment:
Should we say `visit` our support site instead of `review` our support site?

Reply:
Btw @maxxcrawford, keep in mind that changing the key means the site will show ??? when this is merged until the PR that points to the new key is merged too.

Reply:
We are using "visit". I also restored the previous string.